docs/rules/no-extra-parens.md

```diff
@@ -20,7 +18,7 @@ This rule takes 1 or 2 arguments. The first one is a string which must be one of
 
 The second one is an object for more fine-grained configuration when the first option is "all".
 
-#### "all"
+### "all"
```

Comment:
This is probably for the next pass, but it looks like in most place we use code brackets for options as in `all` and not double quotes.